packages/core/src/components/region/RegionFilterButton/RegionFilterButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { useRef } from 'react'
+
+import {
+  Button as UIButton,
+  Icon as UIIcon,
+  useUI,
+  regionSliderTypes,
+} from '@faststore/ui'
+
+type RegionFilterButtonProps = {
+  filterByPickupPoint?: {
+    enabled?: boolean
+    label?: string
+    icon?: {
+      icon?: string
+      alt?: string
+    }
+  }
+}
+
+function RegionFilterButton({
+  filterByPickupPoint: {
+    enabled: filterByPickupPointEnabled,
+    label: filterByPickupPointLabel,
+    icon: { icon: filterByPickupPointIcon, alt: filterByPickupPointAlt },
```

Comment:
```suggestion
    icon: { icon: filterByPickupPointIcon, alt: filterByPickupPointAlt } = {},
```

preventing the TypeError when no object